test/unit_test/test_common.h

```diff
@@ -33,3 +33,10 @@ void check_collections_exact(const ContainerOneT<T>& lhs, const ContainerTwoT<T>
 {
   BOOST_CHECK_EQUAL_COLLECTIONS(std::begin(lhs), std::end(lhs), std::begin(rhs), std::end(rhs));
 }
+
+template <typename T>
+void check_namespaces_exact(const std::vector<std::vector<T>>& lhs, const std::vector<std::vector<T>>& rhs) {
```

Comment:
The implementation of this has nothing to do with namespaces, could the name of the function be changed to reflect the generic nature?

Reply:
I've changed it to `check_vector_of_vectors_exact` now. Is that ok?